esp/services/esdl_svc_engine/esdl_svc_custom.cpp

```diff
@@ -306,13 +306,46 @@ void CEsdlCustomTransform::processTransform(IEspContext * context, StringBuffer
         auto user = context->queryUser();
         if (user)
         {
+            static const std::map<SecUserStatus, const char*> statusLabels =
+            {
+#define STATUS_LABEL_NODE(s) { s, #s }
+                STATUS_LABEL_NODE(SecUserStatus_Inhouse),
+                STATUS_LABEL_NODE(SecUserStatus_Active),
+                STATUS_LABEL_NODE(SecUserStatus_Exempt),
+                STATUS_LABEL_NODE(SecUserStatus_FreeTrial),
+                STATUS_LABEL_NODE(SecUserStatus_FreeTrial),
```

Comment:
This should be SecUserStatus_csdemo, not a second occurrence of SecUserStatus_FreeTrial